easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-5.6.0-foss-2019b-METIS-5.1.0.eb

```diff
@@ -0,0 +1,26 @@
+name = 'SuiteSparse'
+version = '5.6.0'
+local_metis_ver = '5.1.0'
+versionsuffix = '-METIS-%s' % local_metis_ver
+
+homepage = 'https://faculty.cse.tamu.edu/davis/suitesparse.html'
+description = """SuiteSparse is a collection of libraries manipulate sparse matrices."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'unroll': True, 'pic': True}
+
+source_urls = ['https://github.com/DrTimothyAldenDavis/SuiteSparse/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['76d34d9f6dafc592b69af14f58c1dc59e24853dcd7c2e8f4c98ffa223f6a1adb']
+
+builddependencies = [('CMake', '3.15.3')]
+
+dependencies = [('METIS', local_metis_ver)]
+
+parallel = 1
+
+#prebuildopts = "sed -i 's/-openmp/-fopenmp/g' SuiteSparse_config/SuiteSparse_config.mk && "
```

Comment:
If this is not really needed any longer it should be removed.
If it needs to stay as a reminder it should be "# prebuildopts..." not the " ". This is only a style issue, but needs to be fixed.